sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/cloud/service/implementation/identity/StaticAccessTokenCache.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.service.implementation.identity;
+
+import com.azure.core.credential.AccessToken;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Implementation of {@link Cache} that provide static cache.
+ */
+public class StaticAccessTokenCache implements Cache<String, AccessToken> {
```

Comment:
we already know we're caching `AccessToken`, shall we validate the expiration of the `AccessToken` when putting it into cache or returning it to user?

Reply:
we validate it [here.](https://github.com/Azure/azure-sdk-for-java/blob/e24e20aece1de8ada9222c3f1ce3a4e226023496/sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/cloud/service/implementation/identity/credential/CacheableTokenCredential.java#L29)

Reply:
yeah, I think making it here would make it more coherence?

let's say when a cache returns you something, wouldn't it be nice that thing is valid?